Makefile

```diff
@@ -1,37 +1,41 @@
-# mark targets to avoid confusion with files
-.PHONY: clean dir help java node
-all: clean java node
-
-JAVA_DIR = ./java/src/main/gen
-PROTO_DIR = ./api
+DIR = $(shell pwd)
+JAVA_DIR = ./java/src/main/java
 NODE_DIR = ./node
-PROTOC_BIN ?= protoc
-ALL_PROTO_FILES = $(shell find api -iname "*.proto" -exec echo -n '"{}" ' \; | tr '\n' ' ')
 
-dir: ## Creates outdir if necessary
-	[ -d $(JAVA_DIR) ] || mkdir -p $(JAVA_DIR)
+OUTPUT ?= $(JAVA_DIR)
```

Comment:
Java als default? That’s racist. 😉

Reply:
since proto/grpc is language agnostic, we should provide a build script and environment setup which supports generating code for various languages. The current setup for example works for `java`, `js`, `ruby` and others just be specifying `LANGUAGE` and `OUTPUT`. IMHO we should add `TypeScript` and `go` to this list (by enhancing the Makefile and Docker image) and be good